ExtensionPack/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# C/C++ Extension Pack for Visual Studio Code Change Log
+
+
+## Version 0.1.0: September 1, 2020
+### List of extensions:
+* C/C++ (Microsoft)
+* C/C++ Themes (Microsoft)
+* Cmake Tools (Microsoft)
+* Remote Development extension pack (Microsoft)
+* GitHub Pull Requests and Issues (GitHub)
+* Visual Studio Codespaces (Microsoft)
+* LiveShare Extension Pack (Microsoft)
+
```

Comment:
Can we add Better C/C++ Syntax? https://github.com/jeff-hykin/cpp-textmate-grammar

Reply:
@jureid

Reply:
Let's remove "(Microsoft)" and "GitHub" from the names

Reply:
@sean-mcmanus do you use that extension with ours?

Reply:
Better C/C++ Syntax is a more recent version of the lexical syntax highlighter used by VS Code.